formatTest/unit_tests/expected_output/jsx.re

```diff
@@ -0,0 +1,180 @@
+type component = {displayName: string};
+
+let (\/><) a b => a + b;
+
+let (><) a b => a + b;
+
+let (\/>) a b => a + b;
+
+let tag1 = 5 \/>< 6;
+
+let tag2 = 5 >< 7;
+
+let tag3 = 5 \/> 7;
+
+let module Bar = {
+  let createElement c::c=? children => {
+    displayName: "test"
+  };
+};
+
+let module Nesting = {
+  let createElement children => {
+    displayName: "test"
+  };
+};
+
+let module Much = {
+  let createElement children => {
+    displayName: "test"
+  };
+};
+
+let module Foo = {
+  let createElement a::a=? b::b=? children => {
+    displayName: "test"
+  };
+};
+
+let module One = {
+  let createElement
+      test::test=?
+      foo::foo=?
+      children => {
+    displayName: "test"
+  };
+};
+
+let module Two = {
+  let createElement foo::foo=? children => {
+    displayName: "test"
+  };
+};
+
+let module Sibling = {
+  let createElement
+      foo::foo=?
+      (children: list component) => {
+    displayName: "test"
+  };
+};
+
+let module Test = {
+  let createElement yo::yo=? children => {
+    displayName: "test"
+  };
+};
+
+let module So = {
+  let createElement children => {
+    displayName: "test"
+  };
+};
+
+let module Foo2 = {
+  let createElement children => {
+    displayName: "test"
+  };
+};
+
+let module Text = {
+  let createElement children => {
+    displayName: "test"
+  };
+};
+
+let module Exp = {
+  let createElement children => {
+    displayName: "test"
+  };
+};
+
+let module Pun = {
+  let createElement intended::intended=? children => {
+    displayName: "test"
+  };
+};
+
+let module Namespace = {
+  let module Foo = {
+    let createElement
+        intended::intended=?
+        children => {
+      displayName: "test"
+    };
+  };
+};
+
+let module LotsOfArguments = {
+  let createElement
+      argument1::argument1=?
+      argument2::argument2=?
+      argument3::argument3=?
+      argument4::argument4=?
+      argument5::argument5=?
+      argument6::argument6=?
+      children => {
+    displayName: "test"
+  };
+};
+
+let b = 2;
+
+let selfClosing = <Foo />;
+
+let selfClosing2 = <Foo a=1 b=true />;
+
+let a = <Foo><Bar c=(fun a => a + 2) /></Foo>;
+
+let a3 = <So><Much><Nesting /></Much></So>;
+
+let a4 =
+  <Sibling>
+    <One test=true foo=b />
+    <Two foo=b />
+  </Sibling>;
+
+let a5 = <Foo>"testing a string here"</Foo>;
+
+let a6 =
+  <Foo2>
+    <Text>"testing a string here"</Text>
+    <Test yo=1 />
+    <Text>"another string"</Text>
+    <Bar />
+    <Exp>{2 + 4}</Exp>
+  </Foo2>;
+
+let intended = true;
+
+let punning = <Pun intended />;
+
+let namespace = <Namespace.Foo />;
+
+let c = <Foo />;
+
+let d = <Foo />;
+
+let spaceBefore =
+  <So><Much><Nesting /></Much></So>;
+
+let spaceBefore2 = <So><Much /></So>;
+
+let siblingNotSpaced = <So><Much /><Much /></So>;
+
+/* doesn't work (pretty print messes it up): let jsxInArray = [(<Foo />)]; */
+let testFunc b => b;
+
+let jsxInFnCall = testFunc (<Foo />);
```

Comment:
Can we just consider the JSX block to be a "simple expression" - as if `<Foo` was the opening paren, and `</Foo>` or `/>` was the closing paren?